vm-setup/roles/common/tasks/main.yml

```diff
@@ -68,6 +68,7 @@
                 - ip: "{{ baremetal_network_cidr | nthhost(5) }}"
                   hostnames:
                     - "api"
+                    - "api-int"
```

Comment:
This change is still needed, at least for now - the masters are pointing at dnsmasq when they come up and they get their ignition config from api-int

Reply:
I guess this is fine as an interim solution in the  VM case, but how do we avoid requiring a new DNS record for the API VIP in the baremetal case?  cc @celebdor and @cybertron

Reply:
The solution is to get the VIPs from install config and serve them in coredns (and make ignition fetch from VIP instead of api)

Reply:
https://github.com/openshift-metal3/kni-installer/pull/77 makes api-int available from coredns. We'll still need the change to point ignition at the VIP instead of a DNS name though.

Reply:
If I'm looking in the right place, doesn't ignition happen using HTTPS, verifying the hostname and certificate? The endpoint is also not platform specific: how do we make that work?

https://github.com/openshift/installer/blob/master/pkg/asset/ignition/machine/node.go#L23-L25

Reply:
We also need to tweak the certificate so it includes the VIP in addition to the DNS name. Some work has been done in that direction by other teams and I need to look at getting it integrated into our fork of the installer.